README.md

```diff
@@ -2969,6 +2990,27 @@ staticmethod-decorators = ["staticmethod", "stcmthd"]
 
 ---
 
+### `pycodestyle`
+
+#### [`allow-overlong-lines-for-comment-tags`](#allow-overlong-lines-for-comment-tags)
+
+Whether or not line-length checks (`E501`) should be triggered for
+comments starting with `comment-tags` (by default: ["TODO", "FIXME",
+and "XXX"]).
```

Comment:
I am a little concerned that this option feels quite niche. Can you lay out the reasoning for including it?

Reply:
I extensively use such task tags for task management and use e.g. `git grep TODO` to list all TODO tasks within the repository. To make that convenient I put the task descriptions in a single line so that they are not cut off in the output by `git grep`.

(Sidenote: `git grep` does not support multi-line grepping ... while it's certainly possible to use `ripgrep` to find TODO comments that span multiple lines that does make collaborating with other people more difficult because then they need to install yet another tool to share the workflow).